### ISSUE TITLE
Stopping regular Pull Requests

### ISSUE THREAD

**Issue by FlorianDecker** [SeriousM/WebDriver.IEDriverServer-nuget#19]
Hi @SeriousM !

With the new ChromeDriver [Release Cycle](http://chromedriver.chromium.org/downloads), which releases a ChromeDriver per Chrome Version, we decided internally to change our approach from using Nuget Packages to always download the matching ChromeDriver for the used Chrome version.

This seems to be the approach Chrome wants people to use, given that they reduced the support for each ChromeDriver to the matching Chrome Major Version.  [Here](http://chromedriver.chromium.org/downloads/version-selection) you can find an algorithm on how to find the matching ChromeDriver for a given Chrome version.

Nuget seems to be too static to support a ChromeDriver per Chrome version, given that we dont want to update a Nuget package every 6 week and also need to support more than one Chrome version at a time. 

This new release cycle also broke our automatic update script we used to send you the regular pull requests. Probably just some small fixes needed, but given that we dont use the Nuget package anymore, we decided to not maintain it. If you want, i can share the Powershell script with you.

Thank you for your work on maintaining this Package!

Best Regards,
Florian

CC @MichaelKetting

**Comment by SeriousM** [comment]
Hi @FlorianDecker, thank you so much for sending the pull requests in the last years! I would love to get the powershell script which I will use in a GitHub pipeline (let's see how this works) to automate the package creation.
It was actually a bad idea to from the start to include the binaries in the repository, but it worked quite well in terms of pull requests.

Thank you and all the others again for all the work!

**Comment by FlorianDecker** [comment]
Oh i just saw that this issue is in the IEDriver Repository. I of course meant the ChromeDriver Server.
We will still continue to manually contribute to this Repository if there is some version we need (we had some automation for IEDriver too, but as we dont need the most recent version we will also stop maintaining it).

I have attached the skript we used for ChromeDriver and for InternetExplorerDriver. Dont judge the code too harshly ;) 

I removed some internal details which where just used to make it work on our systems to reduce it to the stuff which is relevant for you. Also, we use ssh authentication for GitHub pushing which you probably have to check.
 
The whole thing should still work (bar authentication) but i have not tested it again. Currently there is a bug in the Function "Assert-Chrome-Version" for the ChromeDriver, as it only checks for Major and Minor Version (which broke because now a 4 part version is used).

[ChromeDriverUpdate.zip](https://github.com/SeriousM/WebDriver.IEDriverServer-nuget/files/3152800/ChromeDriverUpdate.zip)

[InternetExplorerDriverUpdate.zip](https://github.com/SeriousM/WebDriver.IEDriverServer-nuget/files/3152874/InternetExplorerDriverUpdate.zip)

Thanks and good luck with the GitHub Pipeline.
